src/index.html

```diff
@@ -11,7 +11,203 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+    </form>
+    <fieldset class="fieldset">
+      <legend>Personal information</legend>
+      <label
```

Comment:
`<label for="surname" class="label">
    Surname:
    <input`

check all html elements